ntbs-service/DataAccess/ReferenceDataRepository.cs

```diff
@@ -341,9 +342,20 @@ public async Task<IList<PHEC>> GetPhecsByAdGroups(string adGroupsString)
             return await _context.PHEC.Where(phec => adGroups.Contains(phec.AdGroup)).ToListAsync();
         }
 
+        public async Task<IList<Hospital>> GetAllActiveHospitals()
+        {
+            return await _context.Hospital
+                .Where(h => !h.IsLegacy)
+                .OrderBy(h => h.Name)
+                .Include(h => h.TBService)
+                .ThenInclude(tbs => tbs.PHEC)
```

Comment:
You can replace this with the one-liner `.Include(h => h.TBService.PHEC)` if you want